src/App.tsx

```diff
@@ -36,10 +37,13 @@ const App = () => {
         }}
       />
       <Route path="/login" exact>
-        <LoginPage />
+        <MagicLinkLoginPage />
       </Route>
-      <Route path="/link/:linkId" exact>
-        <LinkPage />
+      <Route path="/estonian-id-login" exact>
```

Comment:
I'm wondering if there should just be login, and it decides what to show based on environment?

Reply:
Yup, the plan's to unify once we get the deployment information from the server. Correct, @Tankenstein?

Reply:
Exactly yes, this is temp until that moment.